internal/server/context.go

```diff
@@ -0,0 +1,25 @@
+package server
```

Comment:
`context.go` is not a good name for the file, because `context` has a [special meaning](https://pkg.go.dev/context) and if code stored in file has nothing to do with context then file should have different name.